lblod/frontend-lokaal-mandatenbeheer
# Ben/lmb 427 breadcrumbs + fix routing mandataris pages

## Description

Updated the breadcrumbs + fixed the routing for the mandataris pages.
![Screenshot 2024-05-15 at 15 03 44](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/43848896/32dc49fb-8b4d-422c-8c97-18f5b58d1501)
![Screenshot 2024-05-15 at 15 03 51](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/43848896/74a9342d-43c7-4b1b-97ec-c081039cc92d)
![Screenshot 2024-05-15 at 15 04 02](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/43848896/1eaa677d-9b8d-4201-8a6d-cc41925a4977)

## How to test

Check breadcrumbs, and see if the pages under mandatarissen still work.
The UI is less important, this PR is about the routing.
